src/main/java/core/basesyntax/stategy/AddOperationHandlerImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.stategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.TransactionDto;
+
+public class AddOperationHandlerImpl implements OperationHandler {
+    @Override
+    public void apply(TransactionDto transactionDto) {
+        Fruit fruit = new Fruit(transactionDto.getFruitName());
+        int quantity = transactionDto.getQuantity();
+        Integer oldQuantity = Storage.getDataBase().get(fruit);
+        if (Storage.getDataBase().get(fruit) == null) {
```

Comment:
I guess we should create interface and class which implements that interface to work with storage indirecty. after all create ariable of storage and work with it, insead of Storage.getDataBase().
Do it in other fragments of code too

Reply:
that is class that already implements interface